Make fails

When I run `make` or `make clean all` the build fails:

```
Successfully tagged tvial/docker-mailserver:testing
/bin/bash: -c: line 0: syntax error near unexpected token `fi'
/bin/bash: -c: line 0: `[[ ! -d config.bak ]] && cp -rp config config.bak; fi'
Makefile:19: recipe for target 'backup' failed
```
I'm guessing that we should just remove the `fi` statements? I don't want to do that without asking though as nobody else seems to have encountered this problem.

--- a/Makefile
+++ b/Makefile
@@ -16,8 +16,8 @@ build:
 backup:
 # if backup directories exist, clean hasn't been called, therefore
 # we shouldn't overwrite it. It still contains the original content.
-	@ [[ ! -d config.bak ]] && cp -rp config config.bak; fi
-	@ [[ ! -d testconfig.bak ]] && cp -rp test/config testconfig.bak; fi
+	@ [[ ! -d config.bak ]] && cp -rp config config.bak
+	@ [[ ! -d testconfig.bak ]] && cp -rp test/config testconfig.bak
 
 generate-accounts:
 	@ docker run --rm -e MAIL_USER=user1@localhost.localdomain -e MAIL_PASS=mypassword -t $(NAME) /bin/sh -c 'echo "$$MAIL_USER|$$(doveadm pw -s SHA512-CRYPT -u $$MAIL_USER -p $$MAIL_PASS)"' > test/config/postfix-accounts.cf

